composer.json

```diff
@@ -77,7 +80,6 @@
         "psr/http-server-middleware": "^1.0",
         "s9e/text-formatter": "^2.3.6",
         "symfony/config": "^4.3.4",
-        "symfony/console": "^4.3.4",
```

Comment:
Where is this coming from if it's not explicitly required? We still use the Command class

Reply:
```
illuminate/console   v8.25.0  requires  symfony/console (^5.1.4)
illuminate/database  v8.25.0  requires  symfony/console (^5.1.4)
```

Reply:
If we're using it explicitly, I'd prefer to have the dependency listed explicitly as well.